app/Providers/AppServiceProvider.php

```diff
@@ -29,6 +29,18 @@ public function boot()
         Validator::extend('student_number', function ($attribute, $value, $parameters, $validator) {
             return preg_match('/^(a|pg)[0-9]+$/i', $value) === 1;
         });
+
+        Validator::extend('greater_or_equal_than_field', function ($attribute, $value, $parameters, $validator) {
+            $min_field = $parameters[0];
+            $data = $validator->getData();
+            $min_value = $data[$min_field];
+
+            return $value >= $min_value;
+        });
+
+        Validator::replacer('greater_or_equal_than_field', function ($message, $attribute, $rule, $parameters) {
```

Comment:
Same as in the above comment.
`https://github.com/laravel/framework/blob/adc49acbefe26f03cd8740003d17d7baf154626e/src/Illuminate/Validation/Concerns/ReplacesAttributes.php#L287`